src/model.d.ts

```diff
@@ -1158,6 +1158,12 @@ export interface BulkCreateOptions<TAttributes = any> extends Logging, Transacti
    * Return all columns or only the specified columns for the affected rows (only for postgres)
    */
   returning?: boolean | Array<keyof TAttributes | Literal | Col>;
+
+  /**
+   * Optional override for the conflict fields in the ON CONFLICT part of the query.
+   * Only supported in Postgres >= 9.5 and SQLite >= 3.24.0
+  */
+   conflictFields?: Array<keyof TAttributes>;
```

Comment:
We're not using `fields` anymore in the project, it's slowly being replaced by either `attributes` (if the user must specify the JS name) or `columns` (if it's the DB/sql name)

Based on the typing, I'd propose `conflictAttributes`

Reply:
My only concern there is that we already have this name for `Model.upsert` so that could be confusing.